endpoints/auth.go

```diff
@@ -562,7 +563,8 @@ func CurrentUser(c Context, scopes []string, audiences []string, clientIDs []str
 }
 
 func init() {
-	if appengine.IsDevAppServer() {
+	// TODO(jbd): Replace it with appengine.IsDevAppServer when it's implemented.
+	if os.Getenv("RUN_WITH_DEVAPPSERVER") != "" {
```

Comment:
It has been fixed in the `google.golang.org/appengine` package since then. Check this out:
https://github.com/golang/appengine/blob/b667a5000b082e49c6c6d16867d376a12e9490cd/internal/identity_vm.go#L95